CONTRIBUTING.md

```diff
@@ -9,48 +9,43 @@ Please note that this project is released with a [Contributor Code of Conduct](c
-- New categories or improvements to the existing categorization are welcome.
+- Search previous requests for additions to awesome-hyper before making a new one - your request may be a duplicate.
+- Make an individual pull request for each suggestion to `awesome-hyper`.
+- For Hyper **package** and **theme** names, please use the package name as published on npm, and as displayed in the URI of the package. 
+  - As an example, the title of `https://www.npmjs.com/package/hyper-snazzy` would be `hyper-snazzy`
+- For new suggestions, use the following formatting `[hyper-plugin-name](link) - Your awesome description`
+  - As an example, here's how `hyper-snazzy` would be formatted:
+  - `[hyper-snazzy](https://www.npmjs.com/package/hyper-snazzy) - A super snazzy theme for Hyper, with muted off-neon colors.`
+- Link additions should be added to the **bottom** of the relevant category or sub-category.
+- New categories or improvements to the existing categorization are _always_ welcome.
 - Check your spelling and grammar.
 - Make sure your text editor is set to remove trailing whitespace.
 - The pull request and commit should have a useful title.
-- The body of your commit message should contain a link to the repository.
 
-Thank you for your suggestions!
+Suggestions for ammendments to these guidelines are always welcome - feel free to submit an issue or pull request.
 
-## Creating your own awesome list
+## How to: Add a plugin, theme, or resource to awesome-hyper
 
-To create your own list, check out the [instructions](create-list.md).
+If you have something awesome to contribute to awesome-hyper list, this is how you can do it.
 
-## Adding something to an awesome list
+You'll need to have a GitHub account - you can [register here](https://github.com/join)!
 
-If you have something awesome to contribute to an awesome list, this is how you do it.
-
-You'll need a [GitHub account](https://github.com/join)!
-
-1. Access the awesome list's GitHub page. For example: https://github.com/sindresorhus/awesome
-2. Click on the `readme.md` file: ![Step 2 Click on Readme.md](https://cloud.githubusercontent.com/assets/170270/9402920/53a7e3ea-480c-11e5-9d81-aecf64be55eb.png)
+1. Access awesome-hyper's GitHub page [here](https://github.com/awesome-hyper)
+2. Click on the `README.md` file: ![Step 2 Click on Readme.md](https://cloud.githubusercontent.com/assets/170270/9402920/53a7e3ea-480c-11e5-9d81-aecf64be55eb.png)
 3. Now click on the edit icon. ![Step 3 - Click on Edit](https://cloud.githubusercontent.com/assets/170270/9402927/6506af22-480c-11e5-8c18-7ea823530099.png)
 4. You can start editing the text of the file in the in-browser editor. Make sure you follow guidelines above. You can use [GitHub Flavored Markdown](https://help.github.com/articles/github-flavored-markdown/). ![Step 4 - Edit the file](https://cloud.githubusercontent.com/assets/170270/9402932/7301c3a0-480c-11e5-81f5-7e343b71674f.png)
```

Comment:
This link redirects me to https://help.github.com/categories/writing-on-github/
Maybe add this one instead or one of the sections?